codegen/src/main/java/software/amazon/awssdk/codegen/poet/model/ListSetters.java

```diff
@@ -119,8 +119,12 @@
 
     @Override
     public List<MethodSpec> beanStyle() {
-        MethodSpec.Builder builder = beanStyleSetterBuilder()
-            .addCode(memberModel().isCollectionWithBuilderMember() ? copySetterBuilderBody() : beanCopySetterBody());
+        MethodSpec.Builder builder = beanStyleSetterBuilder();
+        if (memberModel().isCollectionWithBuilderMember()) {
```

Comment:
Because I didn't write the code, I'm not confident that making that change is safe. Can we go with this as-is for now, since this comment is just about a perceived simplification? It'll be refactored with the fix for more complex types in the future.

Reply:
yeah..sure